simmechanics/data/icub2_5/ICUB_2-5_BB_simmechanics_options.yaml.in

```diff
@@ -921,6 +933,26 @@ sensors:
     linkName: l_foot
     sensorName: l_foot_mtb_acc_10b13
     sensorType: "accelerometer"
+  - frameName: SCSYS_R_UPPER_LEG_FT-IMU_3B11
+    exportFrameInURDF: Yes
+    linkName: r_hip_2
+    sensorName: r_upper_leg_ft_imu_3B11
+    sensorType: "imu"
+  - frameName: SCSYS_R_FOOT_FT-IMU_3B14
+    exportFrameInURDF: Yes
+    linkName: r_ankle_2
+    sensorName: r_foot_ft_imu_3B14
+    sensorType: "imu"
+  - frameName: SCSYS_L_UPPER_LEG_FT-IMU_3B12
+    exportFrameInURDF: Yes
+    linkName: l_hip_2
+    sensorName: l_upper_leg_ft_imu_3B12
+    sensorType: "imu"
+  - frameName: SCSYS_L_FOOT_FT-IMU_3B13
+    exportFrameInURDF: Yes
+    linkName: l_ankle_2
+    sensorName: l_foot_ft_imu_3B13
+    sensorType: "imu"
```

Comment:
## exportFrameInURDF -> No
As mentioned in my previous comment aboce, it is not mandatory to export a fake link frame in the URDF. So we can leave `exportFrameInURDF` to the default value "No" (I explain further how the default is set).

## sensorName
I would use all lower case, as for all other sensor names used in the URDF.

## sensorType
For each IMU sensor, we shold add **one accelerometer** and **one gyroscope**. For instance, for SCSYS_L_UPPER_LEG_FT-IMU_3B12:
```
  - frameName: SCSYS_L_UPPER_LEG_FT-IMU_3B12
    linkName: l_hip_2
    sensorName: l_upper_leg_ft_acc_3b12
    sensorType: "accelerometer"
  - frameName: SCSYS_L_UPPER_LEG_FT-IMU_3B12
    linkName: l_hip_2
    sensorName: l_upper_leg_ft_gyro_3b12
    sensorType: "gyroscope"
``` 
The default block is identified by the parser through the `sensorType`...
accelerometer:
https://github.com/robotology/icub-model-generator/blob/44e99c620cbbc237a84929023b80aab220ab0428/simmechanics/data/icub2_5/ICUB_2-5_BB_simmechanics_options.yaml.in#L652-L658
gyroscope:
https://github.com/robotology/icub-model-generator/blob/44e99c620cbbc237a84929023b80aab220ab0428/simmechanics/data/icub2_5/ICUB_2-5_BB_simmechanics_options.yaml.in#L652-L658

The purpose of these default blocks was to avoid to fill the same information for the large number of MTB accelerometers.
the default sensor blob:
https://github.com/robotology/icub-model-generator/blob/44e99c620cbbc237a84929023b80aab220ab0428/simmechanics/data/icub2_5/ICUB_2-5_BB_simmechanics_options.yaml.in#L656-L658
will cause a problem. A workaround would be to set an empty `sensorBlobs:` field, which requires a small change in the parser. **I can help on that.**

## linkName <-- POTENTIAL ISSUE
Is the transform of the IMU sensor frame expressed with respect to the FT parent link, or with respect to the FT sensor frame as illustrated [here](https://github.com/robotology/icub-model-generator/issues/91#issuecomment-412495421)?

Reply:
> ## linkName <-- POTENTIAL ISSUE
> 
> Is the transform of the IMU sensor frame expressed with respect to the FT parent link, or with respect to the FT sensor frame as illustrated [here](https://github.com/robotology/icub-model-generator/issues/91#issuecomment-412495421)?

After discussing with @prashanthr05 and going through the Simmechanics output file, we realized that the relative transform that will appear in the generated URDF model file, under the tag `<sensor><origin>` is computed from the composition of the absolute pose of the IMU sensor frame (defined in the simmechanics output `ICUB_2-5_BB_SIM_MODEL.xml` with respect to the same WORLD frame) and the `linkName` frame pose with respect to the WORLD. Since `l_hip_2` and `r_hip_2` are respectively parents of the FT sensor fixed frames `l_leg_ft_sensor` and `r_leg_ft_sensor`, it is ok to use them as the FT-IMU parent links.